.github/workflows/stale.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   stale:
+    if: ${{ github.repository == 'koishijs/koishi' }}
```

Comment:
is this necessary?

Reply:
Not really but I don't think a fork need this.